.github/workflows/codegen-build-test.yml

```diff
@@ -6,6 +6,9 @@ on:
   workflow_dispatch:
   pull_request:
 
+permissions:
+  id-token: write
+
```

Comment:
Do we need this?  Doesn't look like we use `write` in this job.

Reply:
I'll try removing it